### ISSUE TITLE
Menu option to open project folder (using the native file manager)

### ISSUE THREAD

**Issue by tristanbob** [4ian/GDevelop#3014]
## Description

When working on a project, it is often necessary to add/remove files in the project folder.  Currently, the dev must browse to this folder themselves using a file browser.   This can take many clicks and distract from building the game.

## Solution suggested

- Add a new option in the "File" menu to "Open project folder" or "Browse project folder"

**Comment by 4ian** [comment]
![image](https://user-images.githubusercontent.com/1280130/132141056-0522a58f-ff29-4feb-98bb-c17268601725.png)

There is such a button hidden in the toolbar of the Resources manager!

**Comment by tristanbob** [comment]
Ooops I didn't know that.  I was deleting unused objects in Resource Manager and of course, had to delete them from the file system.  I guess we can close this issue.

It would be nice if the resource manager had an option to delete source files when deleting objects.... is there an issue for that yet?

**Comment by 4ian** [comment]
I don't think there is an issue for anything like that!

**Comment by AlexandreSi** [comment]
I see this feature useful in the resource manager as either:
- A new button "Delete from project and from files" (with better wording)
- A checkbox "Delete from files" not checked by default

on a dialog that appears when:
- Removing a particular resource
- Remove all unused type

Do you see somewhere else this feature should be present?

**Comment by tristanbob** [comment]
I'm not sure where we discussed this (probably Discord) but some concern was mentioned about deleting source files accidentally.  (@Silver-Streak)

Do we want this feature to move "deleted" items to a "Trash" (or similar name) folder?  This protects content from accidental deletion and makes it easy to delete a large number of files manually.

**Comment by Silver-Streak** [comment]
I think that as long as a dialog appears whenever you try to delete something (e.g. double-confirmation), it's **probably** okay?

Although if a trash bin/folder is possible that'd be even safer.

**Comment by AlexandreSi** [comment]
Good idea I think, it would remove a great part of anxiety that deleting files from filesystem generates!
Besides, here is another issue that could maybe be treated at the same time: #816

**Comment by Bouh** [comment]
> I see this feature useful in the resource manager as either: [...] on a dialog that appears when:
> Removing a particular resource
> Remove all unused type

I really don't like the confirmation/alert box but why not for this use case, it's an important and irreversible action 👍
